src/components/ChatInput/styles.tsx

```diff
@@ -5,4 +5,8 @@ export const Container = styled.div`
     justify-content: space-between;
     align-items: center;
     padding: 0% 5%;
+
+    * + * {
```

Comment:
aqui não bastaria somente o seletor "*" ?
```suggestion
    * {
```

Reply:
Ow, não porque eu quero um espaçamento apenas entre os itens da lista. Por isso só adiciono o espaçamento/margin caso haja um item antes. Se não fizesse dessa forma, o primeiro item da lista teria o margin antes.

Reply:
ah, sim! saquei
a prop "`row-gap`" e "`column-gap`" pode ser útil nesse caso que você tá usando flex box

Reply:
Boa demais, já fiz a troca aqui kk.